src/main/java/com/gregtechceu/gtceu/api/registry/registrate/MultiblockMachineBuilder.java

```diff
@@ -92,6 +102,25 @@ protected MultiblockMachineBuilder(Registrate registrate, String name,
                 itemFactory, blockEntityFactory);
     }
 
+    @ApiStatus.Internal
+    public static MultiblockMachineBuilder createKJSMulti(ResourceLocation id) {
+        return new MultiblockMachineBuilder(GTRegistration.REGISTRATE, id.getPath(),
+                WorkableElectricMultiblockMachine::new,
+                MetaMachineBlock::new,
+                MetaMachineItem::new,
+                MetaMachineBlockEntity::createBlockEntity);
+    }
+
+    @ApiStatus.Internal
+    public static MultiblockMachineBuilder createKJSMulti(ResourceLocation id,
+                                                          KJSTieredMachineBuilder.CreationFunction<? extends MultiblockControllerMachine> machine) {
+        return new MultiblockMachineBuilder(GTRegistration.REGISTRATE, id.getPath(),
+                machine::create,
+                MetaMachineBlock::new,
+                MetaMachineItem::new,
+                MetaMachineBlockEntity::createBlockEntity);
+    }
+
```

Comment:
Could we move these into the KJS package using a call to `createMulti` which is public but functionally identical to the constructor?

Reply:
sure